packages/types/config/render.d.ts

```diff
@@ -7,20 +7,65 @@
  *                https://github.com/jshttp/etag#readme
  */
 
+import { IncomingMessage, ServerResponse } from 'http'
```

Comment:
Probably relevant to move this line down under `etag` imports ones (alphabetical)

Reply:
I think because it's a node built-in, it should be at top.

Reply:
Yeah tha'ts right